.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: java
 script:
-- ./gradlew check
+  - ./gradlew check --info
```

Comment:
I think so, yes - because [+travis run gradle with --info; otherwise we don't get any meaningful stacktraces out of the travis test runs](https://github.com/ktoso/reactive-streams/commit/00c7a51320fd6d493e107502f68f46b72df618ca). Without it we get failures on travis which are as helpful as "there was a failure", with `--info` we get a stacktrace of the failure.